packages/react-reconciler/src/ReactFiberPerformanceTrack.js

```diff
@@ -54,6 +54,44 @@ export function logComponentRender(
     return;
   }
   if (supportsUserTiming) {
+    let selfTime: number = (fiber.actualDuration: any);
+    for (let child = fiber.child; child !== null; child = child.sibling) {
+      selfTime -= (child.actualDuration: any);
```

Comment:
I don't love this loop for every rendered component. We could maybe change how we keep track of this but that might require another field. We could also maybe keep track of this by reusing the commit loop.